packages/table/preview/index.tsx

```diff
@@ -94,15 +104,34 @@ const ROW_COUNTS = [
     1000000,
 ];
 
+const CELL_CONTENTS = [
+    CellContent.EMPTY,
+    CellContent.CELL_NAMES,
+    CellContent.LONG_TEXT,
+] as CellContent[];
```

Comment:
is this cast necessary now that it's an enum? if so, def preferable as type declaration instead of cast.

`const CELL_CONTENTS: CellContent[] = ...`